nomenclature/definitions/variable/technology/power-plant.yaml

```diff
@@ -137,20 +137,32 @@ Inflows|Electricity|Hydro|Reservoir:
   description: Inflows into a reservoir expressed in energy
   unit: MWh
 
+Initial Storage Level|Electricity|Hydro|Pumped Storage:
+   description: Initial energy stored in the pumped-storage hydro unit at first load level.
```

Comment:
What is meant with "at first load level"? Also, for clarification: If this variable is used with hourly timesteps, then the value refers to the stored energy at the beginning of the hour - correct?

Reply:
Right, this variable is used at the beginning of the first hour into hourly timesteps.
I'll change it.